src/main/java/banking/model/Card.java

```diff
@@ -0,0 +1,20 @@
+package banking.model;
+
+public class Card {
```

Comment:
Can you use Lombok for generating getters, constructors?

Reply:
fixed